internal/testhelpers/tagsuite/tagsuite.go

```diff
@@ -8,6 +8,45 @@ import (
 	"github.com/thoas/go-funk"
 )
 
+const (
+	Activate      = "activate"
+	Alternative   = "alternative"
+	Auth          = "auth"
+	CLIDeploy     = "cli-deploy"
```

Comment:
Why does this one have a cli prefix? Consider my OCD triggered :p

Reply:
Because this is used in the `TestUpdateBits` integration test. So the tag is for integration tests that ensure the cli has deployed properly. I didn't add the tag but I think that was @mdrohmann's intention.